solara/components/file_drop.py

```diff
@@ -33,17 +35,15 @@ def FileDrop(
     on_total_progress: Optional[Callable[[float], None]] = None,
     on_file: Optional[Callable[[FileInfo], None]] = None,
     lazy: bool = True,
-):
+) -> ipyvuetify.extra.FileInput:
```

Comment:
```suggestion
)
```
This is not the case, the function does not return a widget, but the type inference will (or should) figure this out, so no need to add